modules/models.py

```diff
@@ -56,7 +56,7 @@ def find_model_type(model_name):
     elif any((k in model_name_lower for k in ['gpt4chan', 'gpt-4chan'])):
         return 'gpt4chan'
     else:
-        config = AutoConfig.from_pretrained(Path(f'{shared.args.model_dir}/{model_name}'))
+        config = AutoConfig.from_pretrained(Path(f'{shared.args.model_dir}/{model_name}'), trust_remote_code=shared.args.trust_remote_code)
```

Comment:
Can you test if this addition is necessary? All this line does is read config.json to determine if the model is a seq2seq or not.

Reply:
Can confirm it  is necessary for  `mosaicml/mpt-1b-redpajama-200b`

Reply:
Thanks, fair enough.